content/prefix-delegation-networks.md

```diff
@@ -0,0 +1,262 @@
+# Prefix Delegation
+
+Prefix delegation is a network address allocation technique where BOSH assigns an entire subnet (prefix) to a VM instead of a single IP address. The VM can then subdivide this prefix to allocate individual addresses to internal components, such as application containers or services.
+
+This approach is particularly valuable for container platforms, where a single VM (Diego cell) may host dozens or hundreds of containers, each requiring its own unique, routable IP address.
+
+!!! Note
+    BOSH prefix delegation support requires BOSH Director `v282.1.0+` and Ubuntu Jammy stemcell `v1.943+`.
+
+## Infrastructure Support
+
+| IaaS Provider | CPI Version            | Status    |
+| ------------- | ---------------------- | --------- |
+| AWS           | bosh-aws-cpi v107.0.0+ | Supported |
+
+---
+
+## Prefix Delegation Concepts
+
+### How Prefix Delegation Works
+
+!!! Important
+    BOSH cannot use prefix delegation networks for its own communication. Therefore, VMs using prefix delegation must also have a primary network (IPv4 or IPv6) for BOSH management traffic.
+
+When you configure prefix delegation, the BOSH Director automatically subdivides your network range into smaller prefixes and assigns one to each VM. The delegated prefix becomes available to release jobs running on that VM through the job specification object, allowing them to allocate individual addresses from the prefix internally.
+
+#### Example: IPv4
+
+Given a cloud config with `range: 10.10.0.0/24` and `prefix: 28`:
+
+```text
+Network: 10.10.0.0/24 (256 addresses)
+├─ 10.10.0.0/28   → Reserved (contains gateway, DNS, etc.)
+├─ 10.10.0.16/28  → VM #1 (16 addresses: .16-.31)
+├─ 10.10.0.32/28  → VM #2 (16 addresses: .32-.47)
+├─ 10.10.0.48/28  → VM #3 (16 addresses: .48-.63)
+└─ ... (up to 16 VMs total)
+```
+
+Each `/28` prefix provides 16 IP addresses (14 usable after network and broadcast addresses).
+
+#### Example: IPv6
+
+Given a cloud config with `range: 2001:db8::/64` and `prefix: 80`:
+
+```text
+Network: 2001:db8::/64 (2^64 addresses)
+├─ 2001:db8::0/80         → Reserved
+├─ 2001:db8:0:0:1::/80    → VM #1 (281 trillion addresses)
+├─ 2001:db8:0:0:2::/80    → VM #2 (281 trillion addresses)
+├─ 2001:db8:0:0:3::/80    → VM #3 (281 trillion addresses)
+└─ ... (up to 65,536 VMs possible)
+```
+
+Each `/80` prefix provides 2^48 addresses—more than enough for any container workload.
+
+!!! warning "AWS IPv6 Prefix Limitation"
+    AWS limits IPv6 prefix delegation to `/80` prefixes.
```

Comment:
it only allows `/80`, not more not less. "limits" sounds like it's still variable.
```suggestion
    AWS only allows IPv6 prefix delegation of `/80` prefixes.
```